26.0.0: Compilation failure: `error: too many arguments to function ‘secinfo_feed_version_status’; expected 0, have 1`

Testbuild of 26.0.0 threws an error for me.

```
==> Starting build()...
CMake Deprecation Warning at CMakeLists.txt:18 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


-- Configuring Greenbone Vulnerability Manager...
-- The C compiler identification is GNU 15.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.4.3")
-- Could NOT find Git (missing: GIT_EXECUTABLE) 
-- Looking for clang-format...
-- Looking for clang-format... /usr/bin/clang-format
-- Install prefix: /usr
-- Log file: -
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Checking for module 'libcjson>=1.7.14'
--   Found libcjson, version 1.7.18
-- Checking for module 'libgvm_base>=22.17'
--   Found libgvm_base, version 22.21.0
-- Checking for module 'libgvm_util>=22.17'
--   Found libgvm_util, version 22.21.0
-- Checking for module 'libgvm_osp>=22.17'
--   Found libgvm_osp, version 22.21.0
-- Checking for module 'libgvm_gmp>=22.17'
--   Found libgvm_gmp, version 22.21.0
-- OPENVASD flag is not enabled
-- Checking for module 'gnutls>=3.2.15'
--   Found gnutls, version 3.8.9
-- Checking for module 'glib-2.0>=2.42'
--   Found glib-2.0, version 2.84.1
-- Checking for module 'libbsd'
--   Found libbsd, version 0.12.2
-- Checking for module 'libical>=1.00'
--   Found libical, version 3.0.20
-- Checking for module 'gpgme'
--   Found gpgme, version 1.24.2
-- Looking for PostgreSQL...
-- Found PostgreSQL: /usr/lib/libpq.so (found version "17.4") 
-- Looking for xsltproc...
-- Looking for xsltproc... /usr/bin/xsltproc
-- Not using libtheia - licensing functions disabled
-- Looking for xmltoman...
-- Looking for xmltoman... /usr/bin/xmltoman
-- Looking for xmlmantohtml... /usr/bin/xmlmantohtml
-- Looking for xsltproc...
-- Looking for xsltproc... /usr/bin/xsltproc
-- Found Doxygen: /usr/bin/doxygen (found version "1.13.2") found components: doxygen missing components: dot
-- Configuring done (7.5s)
-- Generating done (0.1s)
-- Build files have been written to: /tmp/makepkg/gvmd/src/build
make: Entering directory '/tmp/makepkg/gvmd/src/build'
[  2%] Building C object src/CMakeFiles/gvm-pg-server.dir/manage_pg_server.c.o
[  2%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp.c.o
[  2%] Building C object src/CMakeFiles/all-manage-obj.dir/manage.c.o
[  2%] Building C object src/CMakeFiles/all-manage-sql-obj.dir/manage_sql.c.o
[  5%] Building C object src/CMakeFiles/gvm-pg-server.dir/manage_utils.c.o
/tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage.c: In function ‘feed_sync_required’:
/tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage.c:5337:21: error: too many arguments to function ‘secinfo_feed_version_status’; expected 0, have 1
 5337 |   feed_status_ret = secinfo_feed_version_status ("cert");
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~  ~~~~~~
In file included from /tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage.c:59:
/tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage_sql_secinfo.h:184:1: note: declared here
  184 | secinfo_feed_version_status ();
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage.c:5351:21: error: too many arguments to function ‘secinfo_feed_version_status’; expected 0, have 1
 5351 |   feed_status_ret = secinfo_feed_version_status ("scap");
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~  ~~~~~~
/tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage_sql_secinfo.h:184:1: note: declared here
  184 | secinfo_feed_version_status ();
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/all-manage-obj.dir/build.make:79: src/CMakeFiles/all-manage-obj.dir/manage.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:252: src/CMakeFiles/all-manage-obj.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  7%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_base.c.o
[  7%] Linking C shared library libgvm-pg-server.so
[ 10%] Building C object src/CMakeFiles/all-manage-sql-obj.dir/manage_sql_alerts.c.o
[ 12%] Building C object src/CMakeFiles/all-manage-sql-obj.dir/manage_sql_events.c.o
[ 12%] Building C object src/CMakeFiles/all-manage-sql-obj.dir/manage_sql_nvts.c.o
[ 15%] Building C object src/CMakeFiles/all-manage-sql-obj.dir/manage_sql_secinfo.c.o
[ 15%] Built target gvm-pg-server
[ 17%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_configs.c.o
/tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage_sql_secinfo.c:6162:1: error: conflicting types for ‘secinfo_feed_version_status’; have ‘int(const char *)’
 6162 | secinfo_feed_version_status (const char *feed_type)
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage_sql_secinfo.c:33:
/tmp/makepkg/gvmd/src/gvmd-26.0.0/src/manage_sql_secinfo.h:184:1: note: previous declaration of ‘secinfo_feed_version_status’ with type ‘int(void)’
  184 | secinfo_feed_version_status ();
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/all-manage-sql-obj.dir/build.make:135: src/CMakeFiles/all-manage-sql-obj.dir/manage_sql_secinfo.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 17%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_delete.c.o
[ 20%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_get.c.o
[ 22%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_license.c.o
[ 22%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_logout.c.o
[ 25%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_port_lists.c.o
[ 27%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_report_configs.c.o
[ 27%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_report_formats.c.o
[ 30%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_tickets.c.o
[ 32%] Building C object src/CMakeFiles/all-gmp-obj.dir/gmp_tls_certificates.c.o
[ 32%] Built target all-gmp-obj
make[1]: *** [CMakeFiles/Makefile2:284: src/CMakeFiles/all-manage-sql-obj.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
make: Leaving directory '/tmp/makepkg/gvmd/src/build'
```

--- a/src/manage_sql_secinfo.h
+++ b/src/manage_sql_secinfo.h
@@ -181,7 +181,7 @@
 #define SECINFO_COMMIT_SIZE_DEFAULT 0
 
 int
-secinfo_feed_version_status ();
+secinfo_feed_version_status (const char *);
 
 pid_t
 manage_sync_scap (sigset_t *);

